### PR TITLE
[Services] Add telegram API

### PR DESCRIPTION
**Pull request by Herklos**
Related to https://github.com/Drakkar-Software/OctoBot-Services/pull/97

Screenshot from 2021-04-09 20-12-22

### CODE REVIEW

File: Services/Services_bases/telegram_api_service/telegram_api.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+
+class TelegramApiService(services.AbstractService):
+    LOGGERS = ["TelegramApiService.client.updates", "TelegramApiService.extensions.messagepacker",
+               "TelegramApiService.network.mtprotosender", "telethon.crypto.aes", "telethon.crypto.aesctr"]
+
+    def __init__(self):
+        super().__init__()
+        self.telegram_client: telethon.TelegramClient = None
+        self.me = None
```
Comment (GuillaumeDSM) [inline Services/Services_bases/telegram_api_service/telegram_api.py:35]: I think we should name this attr in a more explicit way (to know what it is)

Comment (Herklos) [reply Services/Services_bases/telegram_api_service/telegram_api.py:35]: It's "me" :laughing:

File: Services/Services_bases/telegram_api_service/telegram_api.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+               and services_constants.CONFIG_TELEGRAM_API in config[services_constants.CONFIG_CATEGORY_SERVICES]
+
+    def has_required_configuration(self):
+        return services_constants.CONFIG_CATEGORY_SERVICES in self.config \
+               and services_constants.CONFIG_TELEGRAM_API in self.config[services_constants.CONFIG_CATEGORY_SERVICES] \
+               and self.check_required_config(
+            self.config[services_constants.CONFIG_CATEGORY_SERVICES][services_constants.CONFIG_TELEGRAM_API]) \
+               and self.get_is_enabled(self.config)
+
+    async def send_message(self, content, markdown=False, reply_to_message_id=None) -> telegram.Message:
```
Comment (GuillaumeDSM) [inline Services/Services_bases/telegram_api_service/telegram_api.py:129]: when is this used ?

Comment (Herklos) [reply Services/Services_bases/telegram_api_service/telegram_api.py:129]: It's not used for now, do you think we should remove it ?

**Review by GuillaumeDSM — APPROVED** [review]
👍

**Pull request merged**